docs/dependencies.md

```diff
@@ -107,6 +109,10 @@ module.exports = {
 
 See [`script_phase` options](https://www.rubydoc.info/gems/cocoapods-core/Pod/Podfile/DSL#script_phase-instance_method) for a full list of available object keys.
 
+#### platforms.ios.buildTypes
+
+An array of build configurations which will include the dependency. If the array is empty, your dependency will be installed in all configurations. For more details, see [`build configurations`](https://guides.cocoapods.org/syntax/podfile.html#pod).
```

Comment:
We could probably suggest here that if you're working on a helper, you should set `debug` as a value, to avoid shipping it as a part of the release process. 

I imagine some libraries may want to do it by default.